File: app/src/main/java/com/tarento/checkin/checkin/UnsuccessfulLoginActivity.java
package com.tarento.checkin.checkin;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

public class UnsuccessfulLoginActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_unsuccessful_login);
    }

    public void callAdmin(View view) {
        Intent intent = new Intent(this, AdminCheckIn.class);
        startActivity(intent);
    }

    public void tryAgain(View view) {
        finish();
    }
}

File: app/src/main/java/com/tarento/checkin/checkin/UnsuccessfulLoginActivity.kt
package com.tarento.checkin.checkin

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class UnsuccessfulLoginActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_unsuccessful_login)
    }

    fun callAdmin(view: View) {
        val intent = Intent(this, AdminCheckIn::class.java)
        startActivity(intent)
        finish()
    }

    fun tryAgain(view: View) {
        finish()
    }
}
